docs/Developers/Newcomers_Guide.rst.orig

```diff
@@ -0,0 +1,596 @@
+Welcome to the Newcomers' Guide!
```

Comment:
please `git rm docs/Developers/Newcomers_Guide.rst.orig` , `git commit --amend -a` , and then re-push with `--force` enabled